src/js/pre-need/config/form.js

```diff
@@ -170,7 +182,9 @@ const formConfig = {
                         'placeOfBirth',
                         'maritalStatus',
                         'militaryStatus'
-                      ], veteran.properties)
+                      ], _.set('militaryStatus.enum', [
+                        'V', 'R', 'A', 'E', 'S', 'O', 'X'
```

Comment:
This is to remove statuses involving death from the options that a presumably living veteran is filling out for themselves.